lib/SmlMessageBody.js

```diff
@@ -23,139 +23,109 @@ var SmlSetProcParameterRequest = require('./messages/SmlSetProcParameterRequest'
 var SmlSetProcParameterResponse = require('./messages/SmlSetProcParameterResponse');
 
 function SmlMessageBody(messageTag){
-	
+
 	switch(messageTag) {
 		case Constants.PUBLIC_OPEN_REQUEST:
 			return new SmlPublicOpenRequest();
-			break;
```

Comment:
Why you remove the break statement at all? As per of w3school this will stop the execution of more code and case testing inside the block. A break can save a lot of execution time because it "ignores" the execution of all the rest of the code in the switch block.